cli/root.go

```diff
@@ -139,5 +139,17 @@ Start a DefraDB node, interact with a local or remote node, and much more.
 		"Path to the private key for tls",
 	)
 
+	cmd.PersistentFlags().String(
+		"keyring-path",
+		"keys",
+		"Path to store encrypted keys",
+	)
+
+	cmd.PersistentFlags().Bool(
+		"no-keyring",
+		false,
```

Comment:
question: By default we want the keyring?

question: For my understanding, what happens to the flow for using keys when there is no keyring? Does it make sense to make weather keyring is disabled or active based on weather acp is on or off?

Reply:
I think enabled by default is a sensible option. The keyring also stores the peer key and encryption key. If you disable the keyring the peer key will be ephemeral and encryption disabled. I don't think the keyring should be dependent on ACP because of the other keys.

Reply:
Makes sense, would be nice to document this with my other `todo` in the README.md